src/models/transactions/accountSet.ts

```diff
@@ -15,13 +15,32 @@ enum AccountSetFlagEnum {
   asfDepositAuth = 9,
 }
 
+export enum AccountSetTransactionFlags {
+  tfRequireDestTag = 0x00010000,
+  tfOptionalDestTag = 0x00020000,
+  tfRequireAuth = 0x00040000,
+  tfOptionalAuth = 0x00080000,
+  tfDisallowXRP = 0x00100000,
+  tfAllowXRP = 0x00200000,
+}
+
+export interface AccountSetFlagsInterface {
+  tfRequireDestTag?: boolean
+  tfOptionalDestTag?: boolean
+  tfRequireAuth?: boolean
+  tfOptionalAuth?: boolean
+  tfDisallowXRP?: boolean
+  tfAllowXRP?: boolean
+}
+
 export interface AccountSet extends BaseTransaction {
   TransactionType: 'AccountSet'
+  Flags?: number | AccountSetTransactionFlags
```

Comment:
Isn't this supposed to be
```suggestion
  Flags?: number | AccountSetFlagsInterface
```